lib/index.js

```diff
@@ -48,6 +58,43 @@ function activate() {
  */
 module.exports = {
   activate,
-  config: {},
-  provideLinter: linter
+  provideLinter: linter,
+  config: {
```

Comment:
Use a `configSchema` in `package.json` ([example](https://github.com/AtomLinter/linter-stylelint/blob/v3.1.0/package.json#L33)) so Atom can know about the configuration without having to run the package code.